docs/make.jl

```diff
@@ -0,0 +1,22 @@
+using Documenter, PubChem, Catalyst
+
+docpath = Base.source_dir()
+assetpath = joinpath(docpath, "src", "assets")
+cp(joinpath(docpath, "Project.toml"), joinpath(assetpath, "Project.toml"), force = true)
+
+include("pages.jl")
+
+makedocs(sitename = "PubChem.jl",
+         authors = "Samuel Isaacson",
+         format = Documenter.HTML(; analytics = "UA-90474609-3",
+                                  prettyurls = (get(ENV, "CI", nothing) == "true"),
+                                  assets = ["assets/favicon.ico"],
+                                  canonical = "https://LalitChauhan56.github.io/PubChem.jl"),
```

Comment:
This URL is not right. It needs to be updated for what it will be for this repo.